features/step_definitions/user_management/common_steps.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 When "I follow {string} for {user}" do |link_text, user|
-  step %(I follow "#{link_text}" within "#user_#{user.id}")
+  # specifying class because table contains a second edit link on the username
+  step %(I follow "#{link_text}" within "#user_#{user.id} .pf-c-table__action")
```

Comment:
No need for a comment, nor calling another step.
```suggestion
  find("tr#user_#{user.id} .pf-c-table__action").click_link(link_text)
```